### DISCUSSION TITLE
Minimale verplichte velden versus gebruik afwegingen

### DISCUSSION THREAD

**Discussion by marcvanandel** [kadaster-labs/sensrnet-home#287] [Ideas]
Tijdens de FAB sessie op 12 januari 2022 kwam ter sprake welke velden nu verplicht zouden moeten zijn en ook tot welk niveau waardelijsten moeten worden afgedwongen.

Vanuit gebruikersperspectief wil ik zo min mogelijk _moeten_ invullen.

Vanuit het gebruik van de informatie wil ik wel voldoende mogelijkheden hebben. Als er te weinig informatie is ingevoerd, is de kwaliteit onvoldoende voor goed gebruik of informatievoorziening. Als data is ingevoerd die onbekend is en dus 'zomaar wat' is ingevoerd, dan vertroebelt dat ook de kwaliteit en bruikbaarheid.

Wat is een goede afweging? Wat zijn onze ervaringen?

De huidige insteek is het gebruik van waardelijsten in de frontend / user interface geeft steeds meer richting waarin nog veel ruimte zit, terwijl in de API en data opslag vooral geen zaken worden afgedwongen.

**Comment by CelineJansen** [comment]
Dit vind ik een heel sterk iets: "De huidige insteek is het gebruik van waardelijsten in de frontend / user interface geeft steeds meer richting waarin nog veel ruimte zit, terwijl in de API en data opslag vooral geen zaken worden afgedwongen." Op basis van signalen van alternatieve input in de API kan bovendien een behoefte blijken waaraan vervolgens invulling gegeven kan worden binnen de richting die in de frontend gegeven wordt.

**Comment by CelineJansen** [comment]
Voor wat betreft verplichte velden waar nu maar wat ingevuld wordt, omdat de echte waarde onbekend is: bij Gemeente Eindhoven betreft die voornamelijk de hoogte van sensoren. Daar vullen we 999.99 in. (Overigens zijn er ook zat vrijwillige velden die we niet altijd kunnen invullen, maar dat zorgt natuurlijk niet voor vervuiling, die laten we dan gewoon leeg.)

**Reply by marcvanandel** [reply]
Check. Leeg laten wat je niet weet, vind ik beter dan 'dummie waarden' invullen. Tegelijk wordt het dan misschien gemakkelijk om iets niet in te vullen wat je wél weet 😁 Wat is wijsheid? 🤔

**Reply by CelineJansen** [reply]
We zouden ervoor kunnen kiezen om bepaalde waarde waar we meer belang aan hechten een standaard dummie waarde te geven. Dan staat het zichtbaar "fout" voor zowel de gebruiker als de beheerder. Dan kun je dus wel nog signaleren dat een verplicht veld fout is ingevuld (daar kan dan eventueel op gestuurd worden). (Dat heeft toch een andere lading dan wanneer een vrijwillig veld wordt opengelaten).
Dus wel verplicht laten in het informatiemodel, maar sturing bieden voor gebruikers door en suggestie te doen voor een dummie waarde in plaats van iedereen hun eigen verschillende dummies toe te staan?

Ook geen geweldige oplossing, maar dan weten gebruikers in elk geval waar ze aan toe zijn als we geen lege velden toe willen staan...

Welke waarden verplicht moeten zijn zal toch ook vooral uit gebruiksbehoeftes moeten blijken ook.

**Reply by marcvanandel** [reply]
Dat laatste in ieder geval!

Interessante suggestie om een 'voorafingevuld' formulier aan te bieden met standaard 'foutieve maar valide' waarden ... Het lijkt me dat ook dit moet volgen uit gebruikersbehoeftes, niet? Ik zou er geen goede gooi naar kunnen doen ...
